app/filtering.js

```diff
@@ -422,12 +422,13 @@ function registerPermissionHandler (session, partition) {
     // TODO(bridiver) - the permission handling should be converted to an action because we should never call `appStore.getState()`
     // Check whether there is a persistent site setting for this host
     const appState = appStore.getState()
+    const isBraveOrigin = origin.startsWith(`chrome-extension://${config.braveExtensionId}/`)
+    const isPDFOrigin = origin.startsWith(`${pdfjsOrigin}/`)
```

Comment:
minor: this differs slightly from the other usage in [frame.js](https://github.com/brave/browser-laptop/blob/master/app/renderer/components/frame/frame.js#L631) where [`pdfjsOrigin` includes the trailing `/`](https://github.com/brave/browser-laptop/blob/master/app/renderer/components/frame/frame.js#L60)